integration-libs/cdp/root/components/cdp-order/cdp-order.component.html

```diff
@@ -1,103 +1,156 @@
-        </ng-template>
-        <div *ngIf="item.key === 'SHIPPED'; then contentShiped"></div>
-        <ng-template #contentShiped>
-          <span class="cx-consignment-status"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <sapn class="cx-estimated-delivery"
-            >Estimated Delivery: 15, October, 2022</sapn
-          ><br />
-          <span class="cx-tracking-number">Tracking Number: 2495301245</span>
-        </ng-template>
-        <div *ngIf="item.key === 'Cancelled'; then contentCancelled"></div>
-        <ng-template #contentCancelled>
-          <span class="cx-consignment-status"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <h3 style="color: red">Cancellation Processed:</h3>
-        </ng-template>
-        <div
-          *ngIf="item.key === 'DELIVERY_REJECTED'; then deliveryRejected"
-        ></div>
-        <ng-template #deliveryRejected>
-          <span class="cx-consignment-status"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <h3 style="color: red">Delivery Rejected:</h3>
-        </ng-template>
-        <div *ngIf="item.key === 'PICKUP_COMPLETE'; then pickUpComplete"></div>
-        <ng-template #pickUpComplete>
-          <span class="cx-consignment-pickedup"
-            >Picked Up - {{ item.value }} Items</span
-          ><br />
-          <span class="cx-pickedup-detail">Picked Up Date: </span>
-        </ng-template>
-        <div
-          *ngIf="item.key === 'DELIVERY_COMPLETED'; then deliveryComplete"
-        ></div>
-        <ng-template #deliveryComplete>
-          <span class="cx-consignment-status"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <h3 style="color: rgb(19, 18, 18)">Delivery Completed</h3>
-        </ng-template>
-        <div *ngIf="item.key === 'CANCELLED'; then deliveryCancelled"></div>
-        <ng-template #deliveryCancelled>
-          <span class="cx-consignment-cancelled"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <span class="cx-cancelled-detail">Cancellation Processed: </span>
-        </ng-template>
-        <div *ngIf="item.key === 'RETURNED'; then deliveryReturned"></div>
-        <ng-template #deliveryReturned>
-          <span class="cx-consignment-returned"
-            >{{ item.key }} - {{ item.value }} Items</span
-          ><br />
-          <span class="cx-returned-detail">Return Processed: </span>
-        </ng-template>
-      </div>
-      <br />
-    </div>
-    <div class="cx-order-image">
-      <div *ngFor="let prdt of orderImage[order.code]">
-        <div *ngFor="let img of prdt.images; let i = index">
-          <div *ngIf="i < 1">
-            <img [src]="img.url" width="88" height="88" class="slide active" />
+  <div>
+    <ng-container *ngIf="(obser$ | async)?.orders.length; else noOrders">
```

Comment:
Can we reuse `obser$`? instead of `orderValue` in line #4?

Also, please rename `obser$` as `orders$` which is more meaningful